File: Rooms.Application.Abstractions/Events/Notifications/KickNotificationEvent.cs
namespace Rooms.Application.Abstractions.Events.Notifications;

/// <summary>
/// Событие уведомления об исключении пользователя из комнаты.
/// </summary>
public sealed class KickNotificationEvent : TargetedNotificationEvent
{
    /// <summary>
    /// Имя вышедшего зрителя
    /// </summary>
    public required string Name { get; init; }
}

File: Overoom.React/src/services/room-hub/events/notifications/kick-notification.event.ts
import { TargetedNotificationEvent } from './targeted-notification.event.ts';

/** Событие уведомления об исключении пользователя из комнаты. */
export interface KickNotificationEvent extends TargetedNotificationEvent {
  /** Имя вышедшего зрителя */
  name: string;
}
